bluemira/utilities/opt_variables.py

```diff
@@ -519,6 +519,30 @@ def __getitem__(self, name) -> BoundedVariable:
         self._check_presence(name)
         return self._var_dict[name]
 
+    def __getattribute__(self, attr):
+        """
+        Attribute access for variable values
+
+        Parameters
+        ----------
+        attr: str
+            attribute to access
+
+        Returns
+        -------
+        attribute value
+
+        """
+        try:
+            return super().__getattribute__(attr)
+        except AttributeError:
+            try:
+                return self[attr].value
+            except KeyError:
+                raise AttributeError(
+                    f"'{self.__class__.__name__}' object has no attribute '{attr}'"
+                ) from None
+
```

Comment:
What's this doing as part of this PR? Was it necessary for the refactor? Does it change user-facing behaviour?

Reply:
This allows for direct value access of a variable. I thought it was a good idea to have access to both the whole variable by dictionary access and an easy way to get the variables value directly. If you want I can remove it though.